.github/workflows/ci_linux_x64-libshortfin.yml

```diff
@@ -80,32 +80,33 @@ jobs:
     - name: Build shortfin (full)
       run: |
         mkdir ${{ env.LIBSHORTFIN_DIR }}/build
-        cd ${{ env.LIBSHORTFIN_DIR }}/build
         cmake -GNinja \
+          -S${{ env.LIBSHORTFIN_DIR }} \
+          -B${{ env.LIBSHORTFIN_DIR }}/build \
```

Comment:
You can also set `working-directory` for a step (or set a default for an entire job). Running all steps from `shortfin/` could simplify this further.

Reply:
Oh, nice! Refactored accordingly.